trade-finance-manager-api/src/v1/graphql-mappings/mappings/facilities/mapFacility.js

```diff
@@ -0,0 +1,76 @@
+const { formattedNumber } = require('../../../../utils/number');
+const mapFacilityProduct = require('./mapFacilityProduct');
+const mapFacilityType = require('./mapFacilityType');
+const mapFacilityStage = require('./mapFacilityStage');
+const mapFacilityValue = require('./mapFacilityValue');
+const mapBankFacilityReference = require('./mapBankFacilityReference');
+const mapGuaranteeFeePayableToUkef = require('./mapGuaranteeFeePayableToUkef');
+const mapBanksInterestMargin = require('./mapBanksInterestMargin');
+const mapFirstDrawdownAmountInExportCurrency = require('./mapFirstDrawdownAmountInExportCurrency');
+const mapFeeType = require('./mapFeeType');
+const mapFeeFrequency = require('./mapFeeFrequency');
+const mapDates = require('./mapDates');
+const mapUkefExposureValue = require('./mapUkefExposureValue');
+const mapFacilityValueExportCurrency = require('./mapFacilityValueExportCurrency');
+
+const mapFacility = (f, facilityTfm, dealDetails, facilityFull) => {
+  // Ensure facility is valid
+  if (!f) {
+    return null;
+  }
+
+  // Deep clone
+  const facility = JSON.parse(JSON.stringify(f, null, 4));
+
+  const {
+    type,
+    value,
+    facilityStage,
+    guaranteeFeePayableByBank,
+    currency,
+  } = facility;
+
+  facility.ukefFacilityType = type;
```

Comment:
Again -- not sure I like reassigning. here :) 

Also -- Seems a tad odd we're reassigning `ukefFacilityType` to be `type` (and without mapping?)

Reply:
As these are existing and due to the number of changes in this PR, we shall leave this as is